findspam.py

```diff
@@ -31,8 +31,8 @@
 COMMON_MALFORMED_PROTOCOLS = [
     ('httl://', 'http://'),
 ]
-# These types of files frequently gets caught as "misleading link"
-SAFE_EXTENSIONS = ['txt', 'js', 'htm', 'html', 'css', 'php', 'py', 'java', 'rb']
+# These types of files frequently get caught as "misleading link"
+SAFE_EXTENSIONS = set(('txt', 'js', 'htm', 'html', 'css', 'php', 'py', 'java', 'rb'))
```

Comment:
I think you can do `{'txt', 'js', 'htm', 'html', 'css', 'php', 'py', 'java', 'rb'}` here.

Reply:
True. I personally prefer the set constructor syntax since the braces one is easy to mistake for a dictionary